utils/apiObjects.ts

```diff
@@ -0,0 +1,151 @@
+import { hexToDecimal, stringToFelt } from "./feltService";
+import { formatHexString } from "./stringService";
+
+const STARKNET = formatHexString(stringToFelt("starknet"));
```

Comment:
can't you just use `stringToHex` ?

Reply:
You're right, stringToHex is actually like stringToFelt

Reply:
Updated the code to remove stringToFelt and use stringToHex instead
See: https://github.com/starknet-id/app.starknet.id/pull/580/commits/d5edd9286d1336fe6d6637ec84d957315548eca2